### DISCUSSION TITLE
"Initialization failed: Bad JSON escape sequence" when running devgateway.exe

### DISCUSSION THREAD

**Discussion by datahai** [microsoft/Microsoft-Fabric-workload-development-sample#44] [Help] [answered]
Hi,  I've got an issue where I have followed all the necessary steps in the Getting Started tutorial "https://learn.microsoft.com/en-gb/fabric/workload-development-kit/quickstart-sample" and having an issue when running the devgateway.

I get "[Error] Initialization failed: Bad JSON escape sequence: \F. Path 'ManifestPackageFilePath', line 3, position 33."

I have updated all relevant config files with my Entra/Fabric details, successfully build the backend project, and copied the **workload-dev-mode.json** file to C:\

Example of my file:

{  
	"CapacityGuid": "my fabric capacity id", 
	"ManifestPackageFilePath": "C:\Files\ManifestPackage.1.0.0.nupkg",
	"WorkloadEndpointURL": "http://127.0.0.1:5000/workload"
}

Any guidance would be appreciated

**Comment by JosEilersTwyzer** [comment, marked as answer]
Hi datahai, in this case you need to change the ManifestPackageFilePath to "C:\\\\Files\\\\ManifestPackage.1.0.0.nupkg" , I think the problem is in the escaping part.
